"""
練習問題

例外「KeyError」が発生します
try-exceptで例外を補足するようにしてください
例外が発生した場合、以下のように表示します

例外発生:'USA'
"""

nations = {
    "Japan" : "Tokyo",
    "France" : "Paris",
    "Spain" : "Madrid",
}

try :
    print(nations["USA"])
except KeyError as e:
    print("例外発生:" + str(e))
